MapboxNavigation/RouteMapViewController.swift

```diff
@@ -924,6 +933,7 @@ protocol RouteMapViewControllerDelegate: NavigationMapViewDelegate, MGLMapViewDe
     func mapViewControllerDidCancelNavigation(_ mapViewController: RouteMapViewController)
     func mapViewController(_ mapViewController: RouteMapViewController, didSend feedbackId: String, feedbackType: FeedbackType)
     func mapViewControllerShouldAnnotateSpokenInstructions(_ routeMapViewController: RouteMapViewController) -> Bool
+    func mapViewController(_ mapViewController: RouteMapViewController, roadNameAt location: CLLocation) -> String?
```

Comment:
Let’s add a documentation comment explaining the return value, in particular what the `nil` and the empty string values mean.

/ref https://github.com/mapbox/mapbox-navigation-ios/pull/1309#discussion_r182639995

Reply:
Cool 👍